backend/api/v2beta1/artifacts.proto

```diff
@@ -0,0 +1,125 @@
+// Copyright 2024 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client";
+package kubeflow.pipelines.backend.api.v2beta1;
+
+import "google/api/annotations.proto";
+import "google/api/httpbody.proto";
+import "google/rpc/status.proto";
+import "google/protobuf/timestamp.proto";
+
+service ArtifactService {
+  // Finds all artifacts within the specified namespace.
```

Comment:
```suggestion
  // Lists all artifacts within the specified namespace.
```

Reply:
I've opted for consistent wording here relative to the rest of the api docs, e.g: 

> Finds all experiments. Supports pagination, and sorting on certain fields.
> Finds all pipelines within a namespace. 

etc. 

[[1]](https://www.kubeflow.org/docs/components/pipelines/reference/api/kubeflow-pipeline-api-spec/)

Reply:
heh. We'll do a mass change at a later date then :joy: